src/helm.hoverProvider.ts

```diff
@@ -40,6 +42,15 @@ export class HelmTemplateHoverProvider implements vscode.HoverProvider {
         }
 
         if (this.notInAction(doc, pos, word)) {
+            try {
+                // have a test against whether the word is a key element in yaml
+                const { matchedNode } = yamlLocator.getYamlElement(doc, pos);
+                if (!yamlUtil.isKey(matchedNode)) {
+                    return Promise.resolve(null);
+                }
+            } catch (ex) {
+                console.log(ex);
+            }
```

Comment:
You mention this is resolving a TODO comment but the diff does not seem to show any TODO comment being removed - am I missing something?